### PR TITLE
Add workspace overview page

### PR DESCRIPTION
**Pull request by wanglam**
### Description

Add workspace overview page

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/workspace/public/components/workspace_overview.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            .pipe(switchMap((response) => (response.success ? of(response.result) : of(null))))
+        : of(null),
+    [workspaces]
+  );
+
+  return useObservable(workspaceObservable);
+};
+
+export const WorkspaceOverview = () => {
+  const currentWorkspace = useCurrentWorkspace();
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/components/workspace_overview.tsx:32]: There is a `currentWorkspace$` in workspaces, not sure if that can be a substitution of useCurrentWorkspace.

File: src/plugins/workspace/public/components/workspace_overview.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      workspaces
+        ? workspaces.client.currentWorkspaceId$
+            .pipe(switchMap((id) => workspaces.client.get(id)))
+            .pipe(switchMap((response) => (response.success ? of(response.result) : of(null))))
+        : of(null),
+    [workspaces]
+  );
+
+  return useObservable(workspaceObservable);
+};
```
Comment (Hailong-am) [inline src/plugins/workspace/public/components/workspace_overview.tsx:29]: how about use `workspaces.client.currentWorkspace$`?

Comment (wanglam) [reply src/plugins/workspace/public/components/workspace_overview.tsx]: Updated.

File: src/plugins/workspace/public/components/workspace_overview.tsx (hunk 1 above)
Comment (wanglam) [reply src/plugins/workspace/public/components/workspace_overview.tsx]: Updated.

**Review by raintygao — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Review by SuZhou-Joe — APPROVED** [review]

**Pull request merged**